tests/integration/differ.py

```diff
@@ -7,17 +7,13 @@
 Used as part of the integration tests.
```

Comment:
This script is copied from AFDKO, so I suggest updating it to the latest version on AFDKO repository.

Reply:
Done